### PR TITLE
feat: add support to custom template path

### PR DESCRIPTION
**Pull request by jgab-net**
feat: add support to custom template path with te property styleTemplates, and the ability to use the .template extension to avoid highlight errors in editors @jgab-net

**NOTE**: please review the readme option `styleTemplates` and the `test/templates` directory.

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -213,12 +213,16 @@ export type CSSOptions = {
...
   /**
    * Set file name
    * https://github.com/jaywcjlove/svgtofont/issues/48#issuecomment-739547189
    */
-  fileName?: string
+  fileName?: string;
+  /**
+  * Setting class name.
+  */
+  className?: string;
```
Comment (jaywcjlove) [inline src/utils.ts:225]: @jgab-net Is this redundant?

File: src/utils.ts [hunk 2]
```diff
@@ -243,6 +247,11 @@ export function copyTemplate(inDir: string, outDir: string, { _opts, ...vars }:
       if (removeFiles.length > 0) {
         await del([...removeFiles]);
       }
+      createdFiles = await Promise.all(createdFiles.map(async (file) => {
+        const changedFileName = file.replace('.template', '');
+        await moveFile(file, changedFileName);
+        return changedFileName;
+      }));
```
Comment (jaywcjlove) [inline src/utils.ts:254]: @jgab-net What is this mainly for?

File: src/utils.ts (hunk 1 above)
Comment (jgab-net) [reply src/utils.ts:225]: sorry, I will remove that

File: src/utils.ts (hunk 2 above)
Comment (jgab-net) [reply src/utils.ts:254]: this is to move the templates generated with the .template extension, for example style.scss.template to style.scss

Comment (jgab-net) [reply src/utils.ts:250]: apparently copy-template-dir can't do it, so I move them, I could filter the array to move only those that have .template, what do you think?

Comment (jaywcjlove) [reply src/utils.ts:250]: @jgab-net I think it needs to be judged, only the files with the suffix `.template` need to be moved.

Comment (jgab-net) [reply src/utils.ts]: ready, I made the change

**Comment by jaywcjlove** [comment]
@jgab-net Upgrade `+ svgtofont@3.7.0`

**Pull request merged**